src/scripts/main.js

```diff
@@ -22,7 +22,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  element.insertAdjacentHTML('beforeend', `<ul></ul>`);
```

Comment:
let`s try to use createElemet in this task)

Reply:
Ok)